### PR TITLE
Revert "Split up install to separate CRs (#5436)"

### PR DESCRIPTION
**Pull request by sdake**
Please see the script in scripts/grab_operator_yamls.sh.
    
This reverts commit 81da298400ddef3c1c1f533c5ecb164f7f6622ee.

### CODE REVIEW

File: static/operator-profile-minimal.yaml [hunk 1]
```diff
@@ -5,6 +5,5 @@ metadata:
   namespace: istio-operator
   name: example-istiocontrolplane
 spec:
-  profile: default
```
Comment (ostromart) [inline static/operator-profile-minimal.yaml]: Is this correct? shouldn't it be minimal?

File: static/operator-profile-sds.yaml [hunk 2]
```diff
@@ -5,6 +5,5 @@ metadata:
   namespace: istio-operator
   name: example-istiocontrolplane
 spec:
-  profile: minimal
```
Comment (ostromart) [inline static/operator-profile-sds.yaml]: This should be sds?

File: static/operator.yaml [hunk 3]
```diff
@@ -252,3 +252,11 @@ spec:
               value: "istio-operator"
 ...
 ---
+apiVersion: install.istio.io/v1alpha2
```
Comment (ostromart) [inline static/operator.yaml]: This should still be removed as it leads to races. The docs should split up operator.yaml and the profile CRs.

Comment (sdake) [reply static/operator.yaml]: Some attention is needed here.  The first step is to revert the problems introduced by manual modification of these files.  There is automated tooling to do this work.

File: static/operator-profile-sds.yaml (hunk 2 above)
Comment (sdake) [reply static/operator-profile-sds.yaml]: the automated tooling does some seding here, but it appears I didn't revert enough PRs - will address.

File: static/operator-profile-minimal.yaml (hunk 1 above)
Comment (sdake) [reply static/operator-profile-minimal.yaml]: the automated tooling does some seding here, but it appears I didn't revert enough PRs - will address.

**Comment by sdake** [comment]
@ostromart if you look in detail at the change, this reverts the name changes - which reverts some of the content.  Please take a look at the generator script if you wish to follow up after this PR merges with corrected automation.

File: static/operator-profile-sds.yaml (hunk 2 above)
Comment (sdake) [reply static/operator-profile-sds.yaml]: this should not be sds, the name of the file is minimal?

Comment (ostromart) [reply static/operator-profile-sds.yaml]: the old file name is minimal but it's being changed back to operator-profile-sds.yaml.

File: static/operator.yaml (hunk 3 above)
Comment (ostromart) [reply static/operator.yaml]: Ok, I'll put this in and redo through the script.

**Pull request merged**